index.js

```diff
@@ -18,7 +18,7 @@ function findRuntime(){
     return runtime;
   }
 
-  runtime = path.join(__dirname, '../../jade/runtime.js');
+  runtime = path.join(__dirname, '../../node_modules/jade/runtime.js');
```

Comment:
I realized, that `require.resolve('jade/runtime')` can be used to find the runtime file.
It would be simpler.